### PR TITLE
instructorFeedbackSessionAdd form: Give an edit links to both sentences #2305

### LINKED ISSUE #2305: instructorFeedbackSessionAdd form: Give an edit links to both sentences
Currently, the edit link is given to the 2nd sentence only. Some users might think the session visibility (first sentence) cannot be changed.
By the way, 'change' may be better than 'edit'? 'change' is the intent, 'edit' is the mechanism.

### PR DESCRIPTION
**Pull request by Nilaksha**
Fixes #2305

### CODE REVIEW

File: src/main/webapp/js/instructorFeedbacks.js [hunk 1]
```diff
@@ -233,20 +233,24 @@ function bindEventsAfterAjax() {
...
 
 function uncommonSettingsButtonClick(){
-    var button = $('#editUncommonSettingsButton');
-    var button_edit = $(button).attr('data-edit');
-    if($(button).text() == button_edit){
+    var button1 = $('#editUncommonSettingsButton1');
+    var button2 = $('#editUncommonSettingsButton2');
+    var button_edit1 = $(button1).attr('data-edit');
+    var button_edit2 = $(button2).attr('data-edit');
+    if(($(button1).text() == button_edit1) || ($(button2).text() == button_edit2)){
```
Comment (kanghj) [inline src/main/webapp/js/instructorFeedbacks.js]: Can you check if this comparison is really required?

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 2]
```diff
@@ -7,7 +7,11 @@ body {
     padding: 60px 0 30px;
 }
 
-a#editUncommonSettingsButton:hover{
+a#editUncommonSettingsButton1:hover{
+    cursor: pointer;
+}
+
+a#editUncommonSettingsButton2:hover{
```
Comment (kanghj) [inline src/main/webapp/stylesheets/teammatesCommon.css]: Maybe use a class instead of duplicating this twice?

**Comment by kanghj** [comment]
Can you post a screenshot of the changes? Also note that the form appears on both instructorFeedbacksPage and instructorFeedbackEditPage, you should make the changes to both pages. Remember to make changes to the tests.

**Comment by Nilaksha** [comment]
I'll do what you have suggested and requested as soon as possible.

**Comment by Nilaksha** [comment]
Instructor Feedback Page
instructorfeedbackspage
 
Instructor Feedback Edit Page
instructorfeedbackeditpage

I made the changes you suggested and pushed the changes into Issue2305 branch. Do I have to make a new pull request? How to make changes to the tests? Any advice?

**Comment by kanghj** [comment]
The pull request now contains many unrelated commits and is difficult to see the relevant changes. You might have pulled master onto your branch, in future you should checkout master before pulling, and then merging master into your branch. Generally, you can continue pushing changes to this pull request, but perhaps in this case you should make a new pull request after you have removed the unrelated commits locally.

One way to see which tests require changing is to run the tests and see which test fails. There will be html comparisons that will fail, those will require you to change the .html files in test/resources/pages.

**Pull request closed without merging**